src/modules/greatAPI.js

```diff
@@ -0,0 +1,12 @@
+import { getLikes, sendLikes } from './getLikes.js';
+
+const hintLikes = (array) => {
+  array.forEach((element) => {
+    element.addEventListener('click', () => {
+      sendLikes(element.id);
+      getLikes(element.id, element);
```

Comment:
[OPTIONAL] Your functions `sendLikes()` and `getLikes()` are running in a synchronous way here and therefore the like counter doesn't update correctly. Consider making this an `async` function and `await` for both functions to finish.